src/ralph/models/xapi/constants.py

```diff
@@ -0,0 +1,32 @@
+"""Constants for xAPI specifications"""
+
+# pylint:disable=invalid-name
```

Comment:
Which one is problematic?

Reply:
Not sure why, but `pylint` complains about `PascalCase naming style` for these constants:
```
Class name "XAPI_ACTIVITY_PAGE" doesn't conform to PascalCase naming style (invalid-name)
Class name "XAPI_VERB_TERMINATED" doesn't conform to PascalCase naming style (invalid-name)
Class name "XAPI_VERB_VIEWED" doesn't conform to PascalCase naming style (invalid-name)
```

Reply:
Can you ignore concerned lines instead of the whole file?